bin/plotting/pycbc_banksim_table_point_injs

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# Copyright (C) 2016 Ian Harry
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+""" Plot effective fitting factor from a bunch of point-source files.
+"""
+
+import sys
+import h5py
+import argparse
+import numpy
+
+import pycbc.version
+from pycbc import results
+
+__author__  = "Ian Harry <ian.harry@astro.cf.ac.uk>"
+__version__ = pycbc.version.git_verbose_msg
+__date__    = pycbc.version.date
+__program__ = "pycbc_banksim_table_point_injs"
+
+parser = argparse.ArgumentParser(usage='',
+    description="Plot effective fitting factor vs mass1 and mass2.")
+parser.add_argument("--version", action="version", version=__version__)
+parser.add_argument('--input-files', nargs='+', default=None, required=True,
+                    help="List of input files.")
+parser.add_argument('--output-file', default=None, required=True,
+                    help="Output file.")
+ 
+opt = parser.parse_args()
+
+col_names = ['Mass 1', 'Mass 2', 'Signal <br\\> recovery <br\\> fraction',
```

Comment:
I agree that this is useful, but I don't want to add more functionality within this pull request. Can we get this merged and then think about adding more information to result pages?